instrumentation/instrumentation.gradle

```diff
@@ -34,16 +34,13 @@ subprojects {
 }
 
 dependencies {
-  implementation project(':instrumentation-api')
-  implementation project(':javaagent-api')
-  implementation(project(':javaagent-tooling')) {
-    exclude module: ':javaagent-bootstrap'
-  }
+  compileOnly project(':instrumentation-api')
+  compileOnly project(':javaagent-api')
+  implementation(project(':javaagent-tooling'))
```

Comment:
Should this also be `compileOnly`?

Reply:
`javaagent` module currently has only `:javaagent-bootstrap` in `shadowInclude` configuration. Which transitively depends on `:javaagent-api` and `:instrumentation-api`. All these three should be added to bootstrap classloader. This is done by `javaagent` module and thus these three modules should NOT come via `instrumentation` module. Everything that comes from the latter goes into `inst` folder inside jar and thus into agent classloader. `javaagent-tooling` should go there as well. Thus it is correct that `instrumentation` module includes it into its own shadowJar.